src/main/java/com/gascalc/model/GasCalcModel.java

```diff
@@ -1,19 +1,39 @@
 package com.gascalc.model;
 
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
 public class GasCalcModel {
 
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	
+	@Column(name="id")
+	private int id;
+	@Column(name="car_make")
 	private String carMake;
+	@Column(name="car_model")
 	private String carModel;
+	@Column(name="car_year")
 	private int year;
+	@Column(name="highway_mpg")
+	private double highwayMPG;
+	@Column(name="city_mpg")
+	private double cityMPG;
```

Comment:
I added id, highwayMPG, and cityMPG to this model so that it properly matched the database table schema I was given for this project. Additionally, I mapped the fields using the @Column tag so that springboot knows which fields to map where. 

Generally, I usually like to name my fields consistently across all layers of the application. This allows consistency in documentation that is often necessary for any apps that span across different code bases (front end, web services, database). For instance, if you make car_year the field for "year" in this model, you will always know what exactly "car_year" is referring to where as otherwise the clarity of "year' may be a bit ambiguous. I didn't change fields here as to not interfere with aspects of your application, but this may be something to consider moving forward.